app/assets/javascripts/hydra-editor/keyboard_support.js

```diff
@@ -1,14 +0,0 @@
-Blacklight.onLoad(function() {
-  $('body').on('keypress', '.multi-text-field', function(event) {
-    var $activeField = $(event.target).parents('.field-wrapper'),
-        $activeFieldControls = $activeField.children('.field-controls'),
-        $addControl = $activeFieldControls.children('.add'),
-        $removeControl = $activeFieldControls.children('.remove');
-    if (event.keyCode == 13) {
-      event.preventDefault();
-      $addControl.click()
-      $removeControl.click()
-    }
-  });
-});
-
```

Comment:
It's the former. Asking in a Yammer group at Penn State who designs or tests for accessibility. I'm looking for a few more people to weigh in and then I'm going to hit up some people individually.